testing/infra/k8s/overlays/local/kustomization.yaml

```diff
@@ -3,6 +3,24 @@ resources:
 - trial-license.yaml
 
 patches:
+- patch: |-
+    apiVersion: apps/v1
+    kind: StatefulSet
+    metadata:
+      name: elastic-operator
+      namespace: elastic-system
+    spec:
+      selector:
+        control-plane: elastic-operator
+      template:
+        spec:
+          containers:
+            - name: manager
+              resources:
+                limits:
+                  memory: "512Mi"
+                requests:
+                  memory: "256Mi"
```

Comment:
We could potentially be even more conservative here. I just took a look at the eck-operator Helm chart, and it requests 150Mi by default: https://github.com/elastic/cloud-on-k8s/blob/b79bb0f644d568f0b082438f0222bf8f2b621ff7/deploy/eck-operator/values.yaml#L35

Reply:
I see, I haven't seen the container go higher than 100MB to be honest. I'll change it